crates/astria-sequencer/src/bridge/bridge_unlock_action.rs

```diff
@@ -60,6 +80,18 @@ impl ActionHandler for BridgeUnlockAction {
             "unauthorized to unlock bridge account",
         );
 
+        let rollup_withdrawal_height = state
+            .get_withdrawal_event_block_for_bridge_account(
+                self.bridge_address,
+                &self.rollup_withdrawal_event_id,
+            )
+            .await
+            .context("failed to get withdrawal event height")?;
+        ensure!(
+            rollup_withdrawal_height.is_none(),
+            "withdrawal event already processed",
+        );
```

Comment:
Maybe useful to include the rollup height in the error?
```suggestion
        if let Some(height) = rollup_withdrawal_height {
            bail!("withdrawal event already processed at rollup height {height}");
        }
```